src/cli/templates/base-config.yaml

```diff
@@ -154,6 +155,7 @@ data_manager:
     sso:
```

Comment:
since `sso` is now just a setting of `links`, can we move these parameters under `links`? Also, can we remove `sso` from `sources` at CLI?